### PR TITLE
Small text should be 14px through all breakpoints

### LINKED ISSUE #1046: <small> font size too small at small viewport
## Issue

The `<small>` tag is [documented in the base styles](https://docs.vanillaframework.io/en/base/typography#small-text) as being a fixed font size of 13px. In the small viewport, this is not the case - it renders as 12.1875px.

I think that a fixed size at all viewports is correct (for example, the "small print" in ubuntu.com's footer is 13px at all viewports).

## Proposed solution

The `<small>` tag should be rendered as 13px at all viewports.

### PR DESCRIPTION
**Pull request by barrymcgee**
Note: @spencerbygraves Please see @yaili's note on #1046 before reviewing.

## Done

Small text should be 14px through all breakpoints

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/small/
- Check text size is 14px through all breakpoints

## Details

Fixes #1046

### CODE REVIEW

File: docs/en/base/typography.md [hunk 1]
```diff
@@ -103,7 +103,10 @@ descriptions.
 
 ## Small text
 
-<small>This is some &lt;small&gt; text with a font size of 13 pixels.</small>
+<a href="https://vanilla-framework.github.io/vanilla-framework/examples/base/small/"
+    class="js-example">
+    View example of the base blockquotes
```
Comment (anthonydillon) [inline docs/en/base/typography.md]: `View example of the small text`

File: examples/base/blockquotes.html [hunk 2]
```diff
@@ -6,4 +6,6 @@
 <blockquote>
   <p><span>“</span> Ubuntu is an ancient African word meaning 'humanity to others'. Ubuntu is an ancient African word meaning 'humanity to others'. <span>”</span></p>
   <p><cite>Canonical</cite></p>
-</blockquote>
\ No newline at end of file
+</blockquote>
+
+<small>this is small text</small>
```
Comment (anthonydillon) [inline examples/base/blockquotes.html]: Remove the extra code here.

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review

**Comment by anthonydillon** [comment]
:+1: just need a design +1 now

**Pull request merged**